assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
-# Additional labels on the namespace do not affect the functionality.
+# They perform the following actions:
+#  - Find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.
+#  - Actively label {orch-namespace} with any workspace.
+# NOTE: It's strongly recommended not to change the value of this property because the {prod-operator-image-name} relies on these labels 
+# and their precise values when reconciling DevWorkspaces. If this configuration is changed, the namespaces isn't automatically
+# recognized by the {prod-operator-image-name} as workspace namespaces unless manually labeled as such by using the default labels and values.
+# Additional labels on the namespace don't affect the functionality.
 che.infra.kubernetes.namespace.labels=app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspaces-namespace
 
-# List of annotations to find {orch-namespace} prepared for {prod-short} users workspaces.
-# Only {orch-namespace} matching the `che.infra.kubernetes.namespace.labels` will be matched against these annotations.
+# List of annotations to find {orch-namespace} prepared for {prod-short} user workspaces.
+# Only {orch-namespace} matching the `che.infra.kubernetes.namespace.labels` is matched against these annotations.
 # {orch-namespace} that matches both `che.infra.kubernetes.namespace.labels` and `che.infra.kubernetes.namespace.annotations`
-# will be preferentially used for User's workspaces.
+# is used preferentially for user workspaces.
 # It's possible to use `<username>` placeholder to specify the {orch-namespace} to concrete user.
-# They are used to:
-#  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.labels`.
-#  - actively annotate {orch-namespace} with any workspace.
+# They perform the following actions:
+#  - Find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.labels`.
+#  - Actively annotate {orch-namespace} with any workspace.
 che.infra.kubernetes.namespace.annotations=che.eclipse.org/username=<username>
 
-# Defines Kubernetes Service Account name which should be specified to be bound to all workspaces Pods.
-# the {prod-short} Operator that Kubernetes Infrastructure will not create the service account and it should exist.
-# OpenShift infrastructure will check if project is predefined(if `che.infra.openshift.project` is not empty):
-#  - if it is predefined then service account must exist there
-#  - if it is 'NULL' or empty string then infrastructure will create new OpenShift project per workspace
-#    and prepare workspace service account with needed roles there
+# Defines Kubernetes Service Account name which should be specified to be bound to all workspace pods.
+# The {prod-short} Operator that Kubernetes infrastructure won't create the service account and it should exist.
```

Comment:
this... isn't a sentence. What should exist... the operator? the service account? 

I'm not sure what we're trying to say here. @tolusha or @skabashnyuk ?